packages/api/tests/endpoints/test-executions.js

```diff
@@ -795,3 +826,107 @@ test.serial('POST /executions/workflows-by-granules returns correct workflows wh
 
   t.deepEqual(response.body.sort(), ['fakeWorkflow', 'workflow2']);
 });
+
+test.serial('POST an api execute record creates a new execution in Dynamo and PG with correct timestamps', async (t) => {
```

Comment:
We need unit tests for the POST endpoint when `arn` is not provided or when the provided execution already exists

Reply:
We also need a basic unit test just to show that the POST endpoint creates the expected execution (with the correct values) in Dynamo and PG. We have a few tests here for the timestamps, but nothing verifying the complete execution record